src/business-logic/services/test/throttled-rundown-service.spec.ts

```diff
@@ -0,0 +1,124 @@
+import { ThrottledRundownService } from '../throttled-rundown-service'
+import { instance, mock, verify, when } from '@typestrong/ts-mockito'
+import { Rundown, RundownInterface } from '../../../model/entities/rundown'
+import { ThrottledRundownException } from '../../../model/exceptions/throttled-rundown-exception'
+
+describe(ThrottledRundownService.name, () => {
+  describe('when two operations are performed within the locked interval', () => {
+    beforeEach(() => jest.useFakeTimers())
+    afterEach(() => jest.useRealTimers())
```

Comment:
Since these are used for all tests, also outside of this describe, you can move this to the outermost describe and remove the duplication :)

Reply:
Fixed